lib/messagebird.js

```diff
@@ -17,13 +17,18 @@ var extend = Object.assign ? Object.assign : require('util')._extend;
  * @param {Integer} timeout
  * @return {Object}
  */
-module.exports = function (accessKey, timeout) {
+module.exports = function (accessKey, timeout, features) {
   var config = {
     accessKey: accessKey,
     timeout: timeout || 5000
+    
   };
 
-
+  var CONVERSATIONSENDPOINT = 'conversations.messagebird.com';
+  if (Array.isArray(features) && features.includes("ENABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX")) {
```

Comment:
Both, `Array.isArray`[¹] and `Array.prototype.includes` [²] are not includes in the supported versions of Node.js of this library (which is according to the README.md v0.10).

```suggestion
  if ('indexOf' in features && features.indexOf('ENABLE_CONVERSATIONSAPI_WHATSAPP_SANDBOX') !== -1) {
```

[¹]: https://node.green/#ES2015-built-ins-Proxy-Array-isArray-support
[²]: https://node.green/#ES2016-features-Array-prototype-includes